# FSRValues.msg
int32 fsr_v
int32 fsr_w
int32 fsr_x
float64 force_v
float64 force_w
float64 force_x